addons/hr_holidays/static/src/components/month_specific_days/month_specific_days.js

```diff
@@ -0,0 +1,40 @@
+import { registry } from "@web/core/registry";
+import { SelectionField, selectionField } from "@web/views/fields/selection/selection_field";
+
+export class MonthSpecificDays extends SelectionField {
+    static props = {
+        ...SelectionField.props,
+        dependFieldName: String,
+    };
+
+    get options() {
+
+        let days = this.props.record.fields[this.props.name].selection;
```

Comment:
Use const instead of let where values don't change
```suggestion
        const days = this.props.record.fields[this.props.name].selection;
```